spec.md

```diff
@@ -148,30 +148,39 @@ that contains both context and data).
 
 ### source
 * Type: Object
-* Description: This describes the software instance that emits the event at
-  runtime (i.e. the producer). It contains sub-properties (listed below)
+* Description: This describes the software where an event occurred. It contains sub-properties (listed below)
 * Constraints:
   * REQUIRED
   * MUST contain at least one non-empty sub-property.
 
 ### source-type
 * Type: String
-* Description: Type of the event source. Providers define list of event
-  sources.
+* Description: The kind of object on which an an event occurred. Providers define list of source-types.
```

Comment:
".. an an.."

Reply:
@inlined you may want to rebase this PR so it just has the new field and not the other PR, otherwise you may continue to get comments that are best addressed in #68

Reply:
My GitHub foo may be lacking. I made this a second PR because the source-id lines will otherwise conflict. That might be better though and I can handle the conflict when both are approved.

Reply:
(@sslavic fixed in #68 thanks)